eventmesh-connectors/eventmesh-connector-chatgpt/src/main/java/org/apache/eventmesh/connector/chatgpt/source/connector/ChatGPTSourceConnector.java

```diff
@@ -150,8 +153,8 @@ private void doInit() {
 
 
     private void validateRequestDTO(ChatGPTRequestDTO bodyObject) {
-        if (bodyObject.getSubject() == null || bodyObject.getDataContentType() == null || bodyObject.getText() == null) {
-            throw new IllegalArgumentException("Attributes 'subject', 'datacontenttype', and 'text' cannot be null");
+        if (bodyObject.getSubject() == null || bodyObject.getText() == null) {
```

Comment:
Since `subject` has a default value, is it still necessary to check the null value of `subject` here? Maybe change it to blank check? If change it to blank check, can `text` also be changed  to blank check?